<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Europe | Map and identify phenotypic mutations in model organism genomes</title>
  <meta property="og:title" content="Map and identify phenotypic mutations in model organism g..." />
  <meta property="og:description" content="The MiModD suite of tools, developed in the
Baumeister la..." />
  <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" />
  <meta name="description" content="The European Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://galaxyproject.eu/posts/2018/03/08/MiModD/plain.html">
  <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Europe" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2018/03/06/Foell-publication-Reproducibleproteomics/plain.html">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">Map and identify phenotypic mutations in model organism genomes</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2018/03/09/propagate-history-hashtags/plain.html">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            

        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              Mar 8, 2018
            </span>

            

            <br />
            <span class="post-tags">
                <span class="label label-success"></span>
              
                <span class="label label-info">tools</span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <p>The MiModD suite of tools, developed in the
<a href="https://celegans.de/mimodd/">Baumeister lab</a> at the Institute for Biology 3
here in Freiburg, is now available on the server.
MiModD is for geneticists who have isolated phenotypic mutants of their
favourite organism, <em>e.g.</em>, through forward genetic screens, and who now wish
to map the underlying mutations and eventually identify them molecularly.</p>

<p>Provided you have followed a mutant phenotype through a cross between the
mutant line and a suitable mapping strain and have sequenced recombinant
offspring genomes, you can use the MiModD tools to identify marker SNPs
(inherited from either parent) and perform a linkage analysis between these
markers and the observed phenotype. You can then continue with MiModD to
identify and annotate candidate mutations that fall into the linked genomic
region.</p>

<p>MiModD can derive linkage patterns from a variety of different crossing
schemes and has been used successfully for species as diverse as the ciliate
<em>Tetrahymena thermophila</em>, the roundworm <em>C. elegans</em>, and zebrafish.</p>

<p>A Galaxy-based tutorial for the tools is available
<a href="https://galaxyproject.github.io/training-material/topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.html">here</a> and there is also a shared
<a href="https://usegalaxy.eu/u/wolfgang-maier/w/identify-phenotypic-mutation-in-arabidopsis">workflow to reproduce the analysis from the tutorial</a>.
Additional example analyses can be found in the
<a href="https://mimodd.readthedocs.io/en/default/tutorial.html">MiModD documentation</a>.</p>

      </div>
    </div>
  </div>



</section>

        </div>
      </div>
    </div>
    





<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>, unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts_plain/2018-03-08-MiModD.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:galaxy@elixir-belgium.org">galaxy@elixir-belgium.org</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-be">usegalaxy-be</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/galaxyproject">galaxyproject</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
