<!DOCTYPE html>
<html lang="en">
  <head>
  <title>Galaxy Belgium | Galaxy and ... Bioconda, CLIP Explorer and RNA workbench @ GCB 2018</title>
  <meta property="og:title" content="Galaxy and ... Bioconda, CLIP Explorer and RNA workbench ..." />
  <meta property="og:description" content="During the German Conference on Bioinformatics in Vienna,..." />
 <!-- <meta property="og:image" content="/assets/media/galaxy-eu-logo.512.png" /> -->
  <meta property="og:image" content="/assets/media/galaxy_project_logo_square.png" />
  <meta name="description" content="Belgian Galaxy Instance">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">




  <link rel="canonical" href="https://usegalaxy.be/posts/2018/09/25/GCB2018_presentations/plain.html">
  <!-- <link rel="shortcut icon" href="/assets/media/galaxy-eu-logo.64.png" type="image/x-icon" /> -->
  <link rel="shortcut icon" href="/assets/media/galaxy_project_logo_square.png" type="image/x-icon" />
  <link rel="alternate" type="application/rss+xml" title="Galaxy Belgium" href="/feed.xml">

  <link href="/assets/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
  <script src="/assets/js/jquery-3.2.1.slim.min.js" integrity="sha256-k2WSCIexGzOj3Euiig+TlR8gA0EmPjuc79OEeY5L45g=" crossorigin="anonymous"></script>
  <script src="/assets/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>
</head>

  <body>
    <div id="wrap">
      <div id="main">
        <div class="container" id="maincontainer">
          




<section id="news-page">
  <div class="row news-header-row">
    
    <a class="col-xs-1 text-center" href="/posts/2018/09/24/galaxy-hts-data-analysis-workshop/plain.html">
      «
    </a>
    
    <div class="col-xs-10 ">
      <div class="text-center">
        <h2 class="title">Galaxy and ... Bioconda, CLIP Explorer and RNA workbench @ GCB 2018</h2>
      </div>
      <hr />
    </div>
    
    <a class="col-xs-1 text-center" href="/posts/2018/09/26/DNS-handover-postmortem/plain.html">
      »
    </a>
    
  </div>
  <div class="row post-metadata">
    <div class="col-md-8 col-md-offset-2">
      <div class="row">
        <div class="col-md-6">
            

        </div>
        <div class="col-md-6 text-right">
          <div>
            <span class="post-date">
              Sep 25, 2018
            </span>

            
            <br />
            <span class="post-location">
                Vienna, Austria
            </span>
            

            <br />
            <span class="post-tags">
                <span class="label label-success"></span>
              
              
            </span>

            
          </div>
        </div>
      </div>
    </div>
  </div>

  <div class="row">
    <div class="col-md-8 col-md-offset-2">
      <div class="news-content">
        <p>During the <a href="https://gcb2018.csb.univie.ac.at">German Conference on Bioinformatics</a> in Vienna, we presented our work related to Galaxy. Read here the abstracts:</p>

<p><strong>CLIP-Explorer: A Galaxy-Pipeline for iCLIP and eCLIP data</strong></p>

<p><em>Florian Heyl, Daniel Maticzka, Rolf Backofen</em> 
Albert-Ludwigs-Universität Freiburg, Germany</p>

<p>RNA is a well-known polymeric molecule which plays a fundamental role in so many regulatory processes like splicing or translation, where RBPs (RNA-binding proteins) take part in. Thousands of RBPs have been found in human cells; some of which have been linked to diseases encompassing various types of cancer. CLIP-Seq (crosslinking immunoprecipitation 
in combination with high-throughput sequencing) facilitated the analysis of RBPs but many protocols like iCLIP or eCLIP require specific tools and parameter sets to analyse the resulting data. These obstacles make the analysis hard to reproduce and demand a tight cooperation with a bioinformatician. CLIP-Explorer resolves these issues by providing a workflow in Galaxy to guide the user to scrutinise iCLIP and eCLIP data, allowing for a user friendly interface to make quick changes to parameters and the pipeline itself. Consequently, CLIP-Explorer is flexible enough to be used for other CLIP-Seq datatypes as well (e.g., PAR-CLIP). The pipeline was tested on eCLIP data of RBFOX2 for which it identified known binding motifs and targets. Three different peak calling algorithms were tested, showing the flexibility of the pipeline and the importance of the parameter and method selection for the overall result.</p>

<p><strong>The RNA workbench</strong></p>

<p><em>Joerg Fallmann (1), Florian Eggenhofer (2)</em>
(1) University of Leipzig, Germany
(2) University of Freiburg, Germany</p>

<p>RNA centric research is of growing importance for medicine and molecular biology. Increasing amounts of data from deep sequencing experiments create a demand for automatic analysis and interpretation solutions. The RNA-Workbench offers a wide range of tools covering classic RNA-bioinformatics as well as RNA-seq fields. Predefined workflows for the annotation of non-coding RNAs or identification of differentially expressed genes are subsets of over 50 included tools from the categories RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling,
RNA-seq analysis and RNA target prediction. RNA specific visualisation solutions for dot-bracket plots and secondary structures are part of the workbench.In contrast to pre-existing solutions, our community driven approach allowes us
to include classic RNA-bioinformatics tools often with the direct support of the tool-authors. These contributions enable us to provide excellent documentation, training material and interactive tours demonstrating the functionality of the
workbench. Building on the Galaxy framework the workbench offers sophisticated analyses to users without command line knowledge, while emphasising reproducibility, customization and effortless scale up to larger infrastructures. The workbench is implemented as Galaxy Docker flavour and therefore easily extendable by additional tools, workflows, tours or training data, that can be installed from the Galaxy ToolShed. The workbench will be further improved and maintained
in an ongoing community effort.</p>

      </div>
    </div>
  </div>



</section>

        </div>
      </div>
    </div>
    





<footer class="navbar-default">
    <div class="container">
        <div class="row">
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <p>All content on this site is available under <a href="https://creativecommons.org/share-your-work/public-domain/cc0/">CC0-1.0</a>, unless otherwise specified.</p>

            </div>
        </div>
        <div class="row">
            <div class="col-lg-12" style="text-align:center">
                <ul class="contact-info">
                    <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-eu/website/tree/master/_posts_plain/2018-09-25-GCB2018_presentations.md">Edit this page on GitHub</a></li>
                    
                      <li><i class="fa fa-envelope"></i><a href="mailto:galaxy@elixir-belgium.org">galaxy@elixir-belgium.org</a></li>
                    
                    
                      <li><i class="fa fa-github"></i><a href="https://github.com/usegalaxy-be">usegalaxy-be</a></li>
                    
                    
                      <li><i class="fa fa-twitter"></i><a href="https://twitter.com/ELIXIRnodeBE">ELIXIRnodeBE</a></li>
                    
                      <li><i class="fa fa-rss"></i>Subscribe <a href="/feed.xml">via RSS (UseGalaxy.eu Feed)</a></li>
                </ul>
            </div>
        </div>
    </div>
</footer>

  </body>
</html>
